### PR TITLE
Changing the framework param to plural since it accepts multiple values

### PR DESCRIPTION
**Pull request by mishra14**
## Bug
Link: https://github.com/NuGet/Home/issues/5854  
Regression: No  
If Regression then when did it last work:   NA
If Regression then how are we preventing it in future:   NA

## Fix
Details: Changes the framework params in add package command to plural since it accepts comma separated values.

## Testing/Validation
Tests Added: No  
Reason for not adding tests:  Not a functional change.
Validation done:  ran `NuGet.CommandLine.XPlat.exe package add -h` locally

I will also fix this in dotnet cli repo.

### CODE REVIEW

**Comment by rohit21agrawal** [comment]
Cc: @rrelyea because this MIGHT be considered a breaking change

**Comment by emgarten** [comment]
Isn't this a breaking change? I would expect to use this by passing the framework parameter multiple times instead of with commas.

**Review by emgarten — CHANGES_REQUESTED** [review]
use multivalue

File: src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/AddPackageReferenceCommand.cs [hunk 1]
```diff
@@ -83,8 +83,8 @@ public static class AddPackageReferenceCommand
                     var packageDependency = new PackageDependency(id.Values[0], VersionRange.Parse(packageVersion));
                     var packageRefArgs = new PackageReferenceArgs(projectPath.Value(), packageDependency, logger)
                     {
-                        Frameworks = MSBuildStringUtility.Split(frameworks.Value()),
-                        Sources = MSBuildStringUtility.Split(sources.Value()),
+                        Frameworks = frameworks.Values.ToArray(),
+                        Sources = sources.Values.ToArray(),
```
Comment (rohit21agrawal) [inline src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/AddPackageReferenceCommand.cs]: will this null ref if its an empty value?

Comment (mishra14) [reply src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PackageReferenceCommands/AddPackageReferenceCommand.cs]: .Values returns an empty list

File: test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatAddPkgTests.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ namespace NuGet.XPlat.FuncTest
     [Collection("NuGet XPlat Test Collection")]
     public class XPlatAddPkgTests
     {
-        private static readonly string projectName = "test_project_addpkg";
+        private static readonly string ProjectName = "test_project_addpkg";
```
Comment (emgarten) [inline test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatAddPkgTests.cs:25]: nit: const

**Review by emgarten — APPROVED** [review]

**Review by jainaashish — APPROVED** [review]
It will be good if we can link this PR to a bug as well for future reference,

**Review by emgarten — APPROVED** [review]
👏

**Pull request merged**